tests/chainerx_tests/unit_tests/routines_tests/test_sorting.py

```diff
@@ -94,3 +127,30 @@ def forward_xp(self, inputs, xp):
         else:
             b = a.argmin(axis)
         return b,
+
+
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize_pytest('input,axis', _count_nonzero_params)
+class TestCountNonzero(op_utils.NumpyOpTest):
+
+    check_numpy_strides_compliance = False
```

Comment:
Could you explain why is it needed?

Reply:
I get the following error for 0-size inputs if it is set to `True`.
```
E       chainer.testing.function_link.FunctionTestError: Strides do not match with NumPy outputs.
E       Expected shapes and dtypes: (2, 0):int64
E       Actual shapes and dtypes:   (2, 0):int64
E       Expected strides: (0, 8)
E       Actual strides:   (8, 8)
```
Since I only use internal routines to reshape, the error is probably arising from `Reshape` which is used in `Sum`. There is a `TODO` in `test_manipulation` to fix strides for 0-size inputs.
(https://github.com/chainer/chainer/blob/9472d08c4fb431a291405af4bd93726e051240ac/tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py#L433-L436)